data/mods/Magiclysm/modinfo.json

```
@@ -4,7 +4,7 @@
     "id": "magiclysm",
     "name": "Magiclysm",
     "authors": [ "KorGgenT", "Aptronym", "LaVeyanFiend" ],
-    "maintainers": [ "Robbietheneko" ],
+    "maintainers": [ "none" ],
```

```suggestion
    "maintainers": [ ],
```

could it be possible to keep `maintainers` empty (if this work without errors?)

---

Very interesting idea, that would probably be better assuming it works without errors yeah.

---

I'll explore this option later once I've slept and have the time. Not like this is an urgent change after all, haha.